apps/sms/js/thread_list_ui.js

```diff
@@ -402,10 +403,18 @@ var ThreadListUI = {
         thread.unreadCount = isRead ? 0 : 1;
         this.mark(thread.id, isRead ? 'read' : 'unread');
 
-        MessageManager.markThreadRead(thread.id, isRead);
+		threadToMark.push(thread.id);
       }
     });
 
+    this.markReadUnread(threadToMark, isRead);
+  },
+
+  markReadUnread: function thlui_markReadUnread(threadToMark, isRead) {
+    threadToMark.forEach((id) => {
+        MessageManager.markThreadRead(id, isRead);
```

Comment:
Ah..got it, there i made mistake. But i don't know why this identation error wasn't coming in my editor.Anyway fix.